### PR TITLE
Feat/migrate scaffolder backend module utils to new backend

### PR DESCRIPTION
**Pull request by david-heidema**
#### :heavy_check_mark: Checklist

- [ ] Added tests for new functionality and regression tests for bug fixes
- [x] Added changeset (run `yarn changeset` in the root)
- [ ] Screenshots of before and after attached (for UI changes)
- [x] Added or updated documentation (if applicable)

### CODE REVIEW

**Comment by jwillaz** [comment]
#1325

**Comment by nia-potato** [comment]
@punkle or @Xantier - Any chance one of you has a moment to review these plugin enhancements for the new backend?

File: plugins/scaffolder-actions/scaffolder-backend-module-utils/src/alpha.ts
Comment (Xantier) [inline plugins/scaffolder-actions/scaffolder-backend-module-utils/src/alpha.ts:1]: Let's name this file something else. It's a bit disingenuous to call this `alpha` if the plan is to deprecate every other option soon enough.

File: .changeset/empty-avocados-jam.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@roadiehq/scaffolder-backend-module-utils': patch
```
Comment (Xantier) [inline .changeset/empty-avocados-jam.md:2]: This should be a minor change here

**Pull request merged**